engine/include/components/rigidbody_component.hpp

```diff
@@ -1,19 +1,47 @@
 ﻿#pragma once
-#include "modules/physics_module.hpp"
+#include "physics_module.hpp"
 
 struct RigidbodyComponent
 {
     // default creates a sphere at 0,2,0
-    RigidbodyComponent(JPH::BodyInterface& bodyInterface)
+    RigidbodyComponent(JPH::BodyInterface& bodyInterface, PhysicsShapes shape = eSPHERE, entt::entity ownerEntity = entt::null)
```

Comment:
This seems incorrect, the owner entity should never be entt::null, why is it a default parameter on the constructor? (seems error prone to me)